include/dt-bindings/clock/mcux_lpc_clock.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_MCUX_LPC_CCM_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_MCUX_LPC_CCM_H_
```

Comment:
Should this file be renamed to reflect `syscon`?

Reply:
Thank you, I have updated this